src/updaterwindow.cpp

```diff
@@ -100,7 +100,7 @@ UpdaterWindow::UpdaterWindow(QWidget *parent)
     updateTitleLabel();
 
     /* Remove window border */
-    setWindowFlags(Qt::Window | Qt::CustomizeWindowHint);
+    setWindowFlags(Qt::CustomizeWindowHint);
```

Comment:
I think this may unset some flags, so instead it should probably be:
```suggestion
    setWindowFlags(windowFlags() | Qt::CustomizeWindowHint);
```